packages/types/src/Metadata/v2/Storage.ts

```diff
@@ -11,6 +11,7 @@ import Null from '../../primitive/Null';
 import Text from '../../primitive/Text';
 import Type from '../../primitive/Type';
 import { MetadataStorageModifier } from '../v1/Storage';
+import { DoubleMapType } from '../v0/Modules';
```

Comment:
tiny nit: we tend to order things alphabetically by default, when there's no order natural ordering

Reply:
Regarding the ordering and spacing rules, can we have them in tslint?

Reply:
That's a good idea. But I think since TypeScript is officially moving towards eslint, we'd like to use eslint everywhere in the mid-term, [this](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) looks for for our case.

But now in this "transition" period before eslint is officially supported, i propose not to overbloat tslint.json, and to manually enforce this few rules.

In any case, you can create an issue on https://github.com/polkadot-js/dev